### DISCUSSION TITLE
Global "downtime" setting.

### DISCUSSION THREAD

**Discussion by neilatuniofyorkspace** [quartznet/quartznet#1462] [Q&A] [answered]
At the moment I have included a time period in all my schedulers' CRON statements to prevent them running during a 24 hourly maintenace cycle; is it possible (I guess/expect not) to include this as a general / central statement which affects all Schedulers?

Many Thanks.

**Comment by lahma** [comment, marked as answer]
You could attach a calendar to your trigger to exclude specific time period.

**Reply by neilatuniofyorkspace** [reply]
Thanks @lahma; 

can this time period be, for example, 4.15 - 4.45 - is this something I could find in your docs ? 
(https://www.quartz-scheduler.net/documentation/quartz-1.x/tutorial/more-about-triggers.html#calendars)

**Reply by lahma** [reply]
The docs you linked have the behavior described. You should probably check the current ICalendar implementations if any of them works for you, from the Quartz repository.
